frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -129,6 +130,8 @@ const PodStatus: React.FC<PodStatusProps> = ({
     outerRadius,
     size,
     standalone,
+    title,
+    subTitle,
```

Comment:
nit: alpha, this should go below `truncSubTitle`

Reply:
arrg, sorry, thanks again!  I hate Mondays :-)